components/appmgt/org.wso2.carbon.appmgt.impl/src/main/java/org/wso2/carbon/appmgt/impl/idp/sso/SSOConfiguratorUtil.java

```diff
@@ -299,9 +299,9 @@ private static String getGatewayUrl(WebApp app) {
         String tenantDomain = PrivilegedCarbonContext.getThreadLocalCarbonContext().getTenantDomain();
 
         if(!tenantDomain.equalsIgnoreCase("carbon.super")){
-            url = url + "/t/" + tenantDomain + app.getContext() + "/" + identifier.getVersion() + "/";
+            url = url + "/t/" + tenantDomain + app.getContext() + "/" ;
         }else {
-            url = url + app.getContext() + "/" + identifier.getVersion() + "/";
+            url = url + app.getContext() + "/" ;
```

Comment:
Formatting errors in line 301, 302("/" ;), 303, 304("/" ;).
Isn't there any constant to carbon.super